cli/cmd/cmd.go

```diff
@@ -376,37 +394,47 @@ func getEndpoint(options config.Options) string {
 	return endpoint
 }
 
-func setLogging(ctx *cli.Context) error {
-	if ctx.Args().First() != "server" {
-		logrus.SetLevel(logrus.WarnLevel)
-		logri.SetLevel(logrus.WarnLevel)
+func setLogging(options *config.Options, ctx *cli.Context, logControl logging.LogControl) error {
+	// Rather than immediately returning from the function on error, keep track
+	// of the errors and continue.  This allows us to process all arguments,
+	// start watchers, etc.
+	var errors []error
+
+	if err := logControl.ApplyConfigFromFile(options.LogConfigFilename); err != nil {
+		errors = append(errors, err)
```

Comment:
If we were planning on maintaining the glog controls permanently then I would say "yes, we need to report all of the errors somehow".  However, since all of these controls are going away with glog, the controls are very rarely used, and the behavior is the same is before these changes (i.e., only reports the first error), I think it is acceptable.  

If you think we need to report them all, I would probably return a list of errors to the calling routine and then report all of them from there.  Something like this:
https://github.com/golang/appengine/blob/master/errors.go#L22-L25